### PR TITLE
Add message `url`

### PR DESCRIPTION
**Pull request by remcohaszing**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

This will be rendered by editors.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ import {list} from './list.js'
 
 // Rules.
 const source = 'retext-contractions'
+const url = 'https://github.com/retextjs/retext-contractions#readme'
```
Comment (ChristianMurphy) [inline index.js:22]: Is this something we could read or construct from the package.json file?

**Review by ChristianMurphy — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (remcohaszing) [reply index.js:22]: Yes, but AFAIK that would add a dependency on NodeJS.

Comment (ChristianMurphy) [reply index.js:22]: I thought that import assertions were supported in browser/deno
https://v8.dev/features/import-assertions
https://deno.com/blog/v1.17#import-assertions-and-json-modules

They are new, so maybe we hold off a bit, but there is a cross platform solution now.

Comment (remcohaszing) [reply index.js:22]: That would be a great solution, but not for now. Not even NodeJS 16 supports it.

The same could be said for `source`, which matches `name` in `package.json`.

Also the value isn’t likely to change, so I don’t think this is a big issue.

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
TS related probably too? Could you check if `4.4.0` solves it?

**Comment by wooorm** [comment]
Released!

**Pull request merged**